docs/content/en/docs/pipeline-stages/status-check.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Deploy Status Checking"
+linkTitle: "Deploy Status Checking"
+weight: 50
+featureId: deploy.status_check
+aliases: [/docs/how-tos/status-check]
+---
+
+This page describes how Skaffold's _deployment status checking_ waits for deployed resources to become ready, and reports errors if they fails to stabilize within a certain time period.
+
+### Overview
+
+Commands that trigger a deployment, like `skaffold dev`, `skaffold deploy`, `skaffold run`, and `skaffold apply`, monitor select Kubernetes resources and wait for them to become ready.
+
+Skaffold monitors the status of the following resource types:
+* [`Pod`](https://kubernetes.io/docs/concepts/workloads/pods/): check that the pod and its containers are in a `Ready` state.
+* [`Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/): check the output of `kubectl rollout status deployment` command 
+* [`Stateful Sets`](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/): check the output of `kubectl rollout status statefulset` command  
+* [`Google Cloud Config Connector resources`](https://cloud.google.com/config-connector/docs/overview): check that the resource state has a `Ready` condition set to `True`.
+
+{{<alert title="Note">}}
+* `status-check` is enabled by default; it can be disabled with the `--status-check=false`
```

Comment:
Replace "`status-check`" with "status checking"?
```suggestion
* Status checking is enabled by default; it can be disabled with the `--status-check=false`
```